### PR TITLE
Added ability to set restSpeedThreshold and restDisplacementThreshold.

### LINKED ISSUE #421: Delay of about 1.3 seconds for functions onRowDidOpen and onRowDidClose after Animation finished
**Describe the bug**
The functions for _ onRowDidOpen_ and _onRowDidClose_ have a delay of about 1.3 seconds after animation visually ends. This leads to wring behavior in my code if the user is fast. There is an issue in react native about it here https://github.com/facebook/react-native/issues/20783 . The suggestion is to use _restSpeedThreshold_ and _restDisplacementThreshold_ in _Animated_. I added _restSpeedThreshold: 100, restDisplacementThreshold: 40,_ to your code at https://github.com/jemise111/react-native-swipe-list-view/blob/master/components/SwipeRow.js#L411 . This got the delay down to about 300ms. So my suggestion is to make _restSpeedThreshold_ and _restDisplacementThreshold_ settable by props just like _tension_ and _friction_ .

**To Reproduce**
1 : Add a function for _onRowDidOpen_ and log Date.now()
2: Add a function for _onRowOpen_ and log Date.now()
3 : swipe left one item (open the right button)
4 : You will see a delay of about 1.3 seconds

**Environment (please complete the following information):**
 - OS: iOS
 - RNSLV Version: 2.2.0
 - RN Version: 0.61.5

### PR DESCRIPTION
**Pull request by benzman81**
This pull request fixes #421

### CODE REVIEW

File: components/SwipeRow.js [hunk 1]
```diff
@@ -412,6 +412,8 @@ class SwipeRow extends Component {
             toValue,
             friction: this.props.friction,
             tension: this.props.tension,
+            restSpeedThreshold: this.props.restSpeedThreshold,
+            restDisplacementThreshold: this.props.restDisplacementThreshold,
```
Comment (jemise111) [inline components/SwipeRow.js]: Do you need to add these to SwipeRow.defaultProps as well?
If they are `undefined` will the animation use the default value or is this problematic?

File: components/SwipeRow.js [hunk 2]
```diff
@@ -584,6 +586,14 @@ SwipeRow.propTypes = {
      * Tension for the open / close animation
      */
     tension: PropTypes.number,
+	/**
```
Comment (jemise111) [inline components/SwipeRow.js]: Can you please run `yarn run fix` on the changes, I think indentation is off here

File: components/SwipeRow.js (hunk 1 above)
Comment (benzman81) [reply components/SwipeRow.js]: Yes, I tested without adding these props to SwipeRow and it still works like before this PR.

File: components/SwipeRow.js (hunk 2 above)
Comment (benzman81) [reply components/SwipeRow.js]: Added commit after yarn run fix.

**Comment by benzman81** [comment]
Fixed and answered ;-)

**Pull request merged**